### PR TITLE
DROOLS-2913: [DMN Designer] Data-types: Grid: FunctionDefinition

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2913

### CODE REVIEW

**Comment by manstis** [comment]
Part of an ensemble:

- https://github.com/kiegroup/appformer/pull/439
- https://github.com/kiegroup/kie-wb-common/pull/2055

File: uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/GridColumnRenderer.java [hunk 1]
```diff
@@ -42,6 +42,21 @@
...
+     * @param headerMetaData MetaData for the header
+     * @param context Contextual information to support rendering
+     * @param headerRowIndex Index of the header row being rendered. Zero based.
+     * @param blockWidth Width of the block of 'equal' header meta data objects.
+     * @param rowHeight Height of the header row.
+     * @return
+     */
+    Group renderHeaderContent(final List<GridColumn.HeaderMetaData> headerMetaData,
+                              final GridHeaderColumnRenderContext context,
+                                     final int headerRowIndex,
```
Comment (karreiro) [inline uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/GridColumnRenderer.java]: I think that this is a code style issue.

**Comment by manstis** [comment]
@karreiro Fixed in new commit (doh!)

**Comment by manstis** [comment]
@jomarko @karreiro This PR is good to review now. Thanks.

**Pull request merged**